projects/docs/docs/commands/plugin.md

```diff
@@ -0,0 +1,46 @@
+---
+slug: '/commands/plugin'
+title: Interact with plugins
+description: 'Learn how to use plugin commands for developing and publishing tuist plugins.'
+---
+
+To help developers with the process of [creating plugins](/plugins/creating-plugins/),
+we provide a set of commands under `tuist plugin`. These plugin commands are aimed at parts of the plugin defined in `Package.swift`.
+
+### Build
+
+`tuist plugin build` will build the plugin.
+
+#### Arguments
+
+| Argument | Short | Description | Default | Required |
+| ------------------ | ----- | ------------------------- | ------- | -------- |
+| `--configuration` | `-c`  | Choose configuration | `debug` | No |
+| `--path`  | `-p`  | Path to the directory that contains the definition of the plugin |  | No |
+| `--build-tests` | n/a | Build both source and test targets | No | No |
+| `--show-bin-path` | n/a | Print the binary output path | No | No |
+| `--targets` | n/a | Build the specificed targets |  | No |
+| `--products` | n/a | Build the specified products | | No |
+
+### Test
+
+`test` subcommand is used for testing the plugin.
+
+#### Arguments
+
+| Argument | Short | Description | Default | Required |
+| ------------------ | ----- | ------------------------- | ------- | -------- |
+| `--configuration` | `-c`  | Choose configuration | `debug` | No |
+| `--path`  | `-p`  | Path to the directory that contains the definition of the plugin |  | No |
```

Comment:
To be consistent with how it's done elsewhere (such as [here](https://docs.tuist.io/commands/generate))

```suggestion
| `--path`  | `-p`  | Path to the directory that contains the definition of the plugin | Current directory | No |
```